example/AsProject/Physical/ARM/X20CP0484/Cpu.sw

```diff
@@ -54,8 +52,6 @@
     <LibraryObject Name="standard" Source="Libraries._AS.standard.lby" Memory="UserROM" Language="binary" Debugging="true" />
     <LibraryObject Name="sys_lib" Source="Libraries._AS.sys_lib.lby" Memory="UserROM" Language="binary" Debugging="true" />
     <LibraryObject Name="AsBrWStr" Source="Libraries._AS.AsBrWStr.lby" Memory="UserROM" Language="binary" Debugging="true" />
-    <LibraryObject Name="MTBasics" Source="Libraries.MTBasics.lby" Memory="UserROM" Language="binary" Debugging="true" />
-    <LibraryObject Name="MTLookUp" Source="Libraries.MTLookUp.lby" Memory="UserROM" Language="binary" Debugging="true" />

```

Comment:
I'm not sure why MTLookUp wasn't added back here. Is it necessary for Intel based systems and not ARM based systems?

Reply:
MTBasics isn't needed for both processors, but MTLookUp is.  MTLookUp should be included in both configs.